articles/virtual-machine-scale-sets/virtual-machine-scale-sets-upgrade-scale-set.md

```diff
@@ -328,6 +331,9 @@ There is one type of modification to global scale set properties that does not f
 
 - You can also use the language-specific [Azure SDKs](https://azure.microsoft.com/downloads/).
 
+> [!NOTE]
+> This action will reimage the selected instance, restoing him to the initial state. The instance may be restarted, and any local data will be lost.
```

Comment:
Please be more specific. We do not use "this" language in notes. Specify exactly what you mean by "this", even if the note is right next to the action you are referring to. Thank you!